snuba/admin/jwt.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
```

Comment:
This is mostly a copy of this GCP example: https://cloud.google.com/python/docs/getting-started/authenticate-users#create_the_source_code

but it seemed reasonable to me